kibana-reports/public/components/context_menu/context_menu.js

```diff
@@ -32,36 +32,18 @@ import {
 
 const replaceQueryURL = () => {
   let url = location.pathname + location.hash;
-  let timeString = url.substring(
-    url.lastIndexOf('time:'),
-    url.lastIndexOf('))')
+  let [, fromDateString, toDateString] = url.match(
+    /time:\(from:(.+),to:(.+?)\)/
```

Comment:
+1

Reply:
Nice, forgot to make this change in context menu too 👍